src/utils/colors.ts

```diff
@@ -74,27 +74,56 @@ function mixPalette(steps: number, color1: Color, color2: Color): Color[] {
  * Like GenColor(4) but with labels
  */
 export const defaultPalette = [
-	new Color(182, 70, 157, t('Purple')),
-	new Color(191, 103, 139, t('Rosy brown'), // TRANSLATORS: A color name for RGB(191, 103, 139)
+	/* eslint-disable @stylistic/function-call-argument-newline -- Allow to put translators comments */
+	new Color(
+		182, 70, 157,
+		t('Purple'),
```

Comment:
Why make it multiline?

Reply:
This one? To make it consistent with the other.
Or all of them? To have the translators comment - its required to be in the same lineas the translation here to be extracted.
See previous - which is even worse as it had mixed newlines in function call